packages/vite/src/node/plugins/manifest.ts

```diff
@@ -58,9 +58,14 @@ export function manifestPlugin(config: ResolvedConfig): Plugin {
         }
 
         if (chunk.imports.length) {
-          manifestChunk.imports = chunk.imports.map((file) =>
-            getChunkName(bundle[file] as OutputChunk)
-          )
+          const imports = []
```

Comment:
```
 manifestChunk.imports = chunk.imports.filter(file => bundle[file]).map((file) =>
            getChunkName(bundle[file] as OutputChunk)
          )
```
This will be more clear. What do you think

Reply:
Agree this will be clear. But it loop twice, or maybe there's not too much of `chunk.imports`?

Reply:
> it loop twice

Agreed.